addons/account/static/src/components/many2one_tax/many2one_tax.js

```diff
@@ -0,0 +1,67 @@
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+
+import { Many2XAutocomplete } from "@web/views/fields/relational_utils";
+
+import { Many2OneField, many2OneField } from "@web/views/fields/many2one/many2one_field";
+
+import { TaxAutoComplete } from "@account/components/tax_autocomplete/tax_autocomplete";
+
+export class Many2OneTaxAutocomplete extends Many2XAutocomplete {
```

Comment:
This class's override functions are 99% the same as `Many2ManyTaxTagsAutocomplete`.
Think of a way to refactor both of them. Don't repeat what's been written